src/gr11review/part1/Review1.java

```diff
@@ -1 +1,75 @@
 package gr11review.part1;
+import java.io.*;
+
+/** 
+* Program that takes the month, converts it to the number of days from the start of the year to the end of the month before it, and adds the to the day of the month the user entered.
+@author Benjamin Teh
+*/
+public class Review1{
+    public static void main(String[] args) throws IOException{
+      BufferedReader keyboard = new BufferedReader(new InputStreamReader(System.in));
+
+      int numMonth = 1;
+      int numDay = 2;
+      int numTotalDay = 0;
+      
+      System.out.println("Enter the month:");
```

Comment:
prompts should be `System.out.print` instead of `System.out.println` so that input can be on the same line as the prompt.

https://github.com/SACHSTech/native-java-and-gr-11-review-assignment-BenjaminTeh/runs/2623634172#step:3:17